share/skel/dot.shrc

```diff
@@ -39,7 +39,7 @@ bind "\\e[1;5C" em-next-word
 bind "\\e[1;5D" ed-prev-word
 alias history='fc -l'
 
-# Fix home/del for mobaxterm
+# try ctrl-v to test keys; fix home/del
```

Comment:
Huh? I think that's a general thing, not confined to this one instance.

Reply:
I've always needed to use this line to make my home and end keys work on bare metal, but it's a different sequence for all my laptops.

What I'm trying to do is make it more obvious that you can type Ctrl-V to see what sequence you need to set it as, for newbies.

Reply:
I'm wondering if that belongs elsewhere, because it works for all the keys.

Reply:
How do you feel about putting the ctrl sequences like siginfo and sigint and ctrl-v in builtin.1?

Reply:
That might be better. We should likely just close this PR and do that instead,. I'll close shortly.